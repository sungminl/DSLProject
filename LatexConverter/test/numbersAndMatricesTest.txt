&a = 412.31231
&b = -123.4
&b = -&a
&c = 512.3 + 23.1
&d = 0.32 - 1331.7
&e = 3124 / 2
&e = 3124 * 2
&f = 2 ^ 5
&g = log(2.3)
&h = sqrt(4)
&i = sin(3.1415926 / 2)
&i = tan(3.1415926 / 2)
&i = cos(3.1415926 / 2)
&a = &b
&a = (2.45)
&a = -(2 + 5) * 1.5
&a = log(2) + 5.32
&b = x
&c = sqrt(4) + &a - log(1) ^ (2 + 5) * sin(25) - cos(&b)

MATRIX
ROW: 10,COL: 10
(&a, &b)
{
&a + 2|
4.255, 1.22 + 4 - 4, &a, 2, 4, 1, 2, 3, 4, 5|
b, c, d, a + 2, b / 2, c ^ 2, 5, 6, a ^ 2, 5 + (4 / &b)|
&a|
&a|
&a|
&a|
&a|
&a|
&a|
}

MATRIX
ROW: 10,COL: 10
(&a, &b)
{
&a
}

