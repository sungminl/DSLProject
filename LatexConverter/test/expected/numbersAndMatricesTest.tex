\documentclass[]{article}
\usepackage{amsmath}
\usepackage{tikz}
\usetikzlibrary{graphs,automata}
\title{Document}
\begin{document}
\[\begin{bmatrix}
 \log( 2.0 ) + ( 5.32 ) + ( 2.0 ) &  \log( 2.0 ) + ( 5.32 ) + ( 2.0 ) &  \log( 2.0 ) + ( 5.32 ) + ( 2.0 ) &  \log( 2.0 ) + ( 5.32 ) + ( 2.0 ) &  \log( 2.0 ) + ( 5.32 ) + ( 2.0 ) &  \log( 2.0 ) + ( 5.32 ) + ( 2.0 ) &  \log( 2.0 ) + ( 5.32 ) + ( 2.0 ) &  \log( 2.0 ) + ( 5.32 ) + ( 2.0 ) &  \log( 2.0 ) + ( 5.32 ) + ( 2.0 ) &  \log( 2.0 ) + ( 5.32 ) + ( 2.0 )\\
4.255 & 1.22 &  \log( 2.0 ) + ( 5.32 ) & 2.0 & 4.0 & 1.0 & 2.0 & 3.0 & 4.0 & 5.0\\
b & c & d & a + ( 2.0 ) & b / ( 2.0 ) & c ^ { 2.0 } & 5.0 & 6.0 & a ^ { 2.0 } & 5.0 + ( 4.0 / ( x ) )\\
 \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 )\\
 \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 )\\
 \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 )\\
 \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 )\\
 \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 )\\
 \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 )\\
 \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 )\\
\end{bmatrix}\]
\[\begin{bmatrix}
 \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 )\\
 \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 )\\
 \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 )\\
 \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 )\\
 \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 )\\
 \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 )\\
 \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 )\\
 \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 )\\
 \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 )\\
 \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 ) &  \log( 2.0 ) + ( 5.32 )\\
\end{bmatrix}\]
\end{document}
